package.json

```diff
@@ -22,12 +22,13 @@
     "lodash-es": "^4.14.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
-    "babel-core": "^6.3.26",
+    "babel-cli": "^6.24.0",
+    "babel-core": "^6.24.0",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-istanbul": "^2.0.1",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.3.16",
     "babel-preset-es2015": "^6.3.13",
+    "babel-preset-es2017": "^6.22.0",
```

Comment:
Yes, otherwise babel can't parse the test file where we use `async` functions, even if they're being used natively.